### DISCUSSION TITLE
Jump Desktop + Rectangle Pro

### DISCUSSION THREAD

**Discussion by conpatricko** [rxhanson/RectanglePro-Community#370] [Q&A]
Okay, loving Rectangle Pro sooo much. But I am often remoting in to my Mac Mini that I have at home and can't get my shortcuts working. I have Rectangle Pro configured on my Macbook Pro the same way I have it configured for my Mac Mini, but there is a key issue I'm having when I am on my MBP and remoting into my Mac Mini:

- NOT WORKING: I have Ctrl + Option + arrow Left/Right key to throw a window to left or right side of screen.
- WORKING: Ctrl + Option + arrow Up/Down arrow key to throw a window to top or bottom half of screen

Most shortcuts work.... but my most used shortcut is Left/Right window throws.

For Left/Right not working, I even disabled using Ctrl + Arrow to switch spaces under Keyboard > Shortcuts > Mission Control because it was doing that when I pressed Ctrl + Option + arrow Left/Right, so I figured it was interfering. I've tried a bunch of different settings in both Jump Desktop and Rectangle Pro.

Any ideas?

**Comment by rxhanson** [comment]
Thanks for reporting. I’m guessing that there must be some sort of keyboard shortcut interception when remoting in. You can view the Rectangle Pro logs by holding the option key in the menu bar menu and selecting view logging, but I suspect we won’t see any logs due to the shortcut being intercepted.

**Reply by MethodFox** [reply]
Out of curiosity, what is your preferred method of remoting in to a computer that is using Rectangle/RectanglePro? 

Let's say I have an iPad w/ Magic Keyboard, or iPhone, and wanted to know some suggestions from you or anyone else about which applications (perhaps Jump Desktop, Screens, AnyDesk, Teamviewer, etc..) have worked for you with hotkeys working?

Thanks!

**Reply by rxhanson** [reply]
Unfortunately, I'm not the right person to ask - I don't use any remoting tools. Maybe someone else in the community will be able to chime in?

**Comment by BrentEubanks** [comment]
This is an old thread, but in case anyone else comes here with a similar issue: I don't have a solution, but I do have a suggestion for troubleshooting.
Karabiner Elements is a key remapping program (https://karabiner-elements.pqrs.org/).  Along with the main app, it has a companion tool called Karabiner EventViewer, which reports whatever keycodes the computer running the app receives.  This is helpful when setting up complex remapping schemes with Karabiner, but could also be useful for troubleshooting situations like the one that the OP is facing.


<img width="1103" alt="image" src="https://github.com/user-attachments/assets/b1e773fd-1752-4b57-a0d7-8fcbe8522bc0" />
